third-party/linshare/src/test/java/org/apache/james/linshare/client/TechnicalAccountCreationRequest.java

```diff
@@ -0,0 +1,65 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.linshare.client;
+
+import org.testcontainers.shaded.com.google.common.annotations.VisibleForTesting;
```

Comment:
Should use the `VisibleForTesting` from guava not testcontainer. (com.google.common.annotations.VisibleForTesting)

Reply:
Should not use it in the first place since we are in src/test

Reply:
Yes, i aware that.